site/docs/foundations/size.mdx

```diff
@@ -5,68 +5,70 @@ sidebar:
+| `--salt-size-base`       | Height only      | 20                     | 28                       | 36                    | 44                      |
+| `--salt-size-selectable` | Height and width | 12                     | 14                       | 16                    | 18                      |
+| `--salt-size-icon`       | Height and width | 12                     | 12                       | 14                    | 16                      |
+| `--salt-size-adornment`  | Height and width | 6                      | 8                        | 10                    | 12                      |
+| `--salt-size-bar`        | Height or width  | 2                      | 4                        | 6                     | 8                       |
+| `--salt-size-indicator`  | Height or width  | 1                      | 2                        | 3                     | 4                       |
+| `--salt-size-border`     | Height or width  | 1                      | 1                        | 1                     | 1                       |
 
 ## Base
 
-`salt-size-base` is the fixed default height for base components in each [density](./density) within the design system. It ensures the consistent alignment of components in horizontal layouts, as well as the vertical alignments of the text values within them. This means the text within each component sits on the same baseline.
+`--salt-size-base` is the fixed default height for base components in each [density](./density) within the design system. It ensures the consistent alignment of components in horizontal layouts, as well as the vertical alignments of the text values within them. This means the text within each component sits on the same baseline.
 
 <ImageSwitcher
   images={[
-    { src: "/img/foundations/size-base.svg", alt: "" },
-    { src: "/img/foundations/size-base-annotated.svg", alt: "" },
+    { src: "/img/foundations/size-base.png", alt: "" },
+    { src: "/img/foundations/size-base-annotated.png", alt: "" },
   ]}
   label="Show sizes"
 />
 
-### Nesting size-base
+### Nesting base
 
 Using size-base, in conjunction with a [spacing token](./spacing) above and below, creates a consistent height for components that can nest a `--salt-size-base` component within them. This provides further standardized heights across the design system.
 
 The following example shows how components can use `--salt-size-base` and `--salt-spacing-50` (top and bottom) to define the height of a component or stackable element, such as a Data Grid row. The height allows a size-base component to be contained within it, but it is not dependent on the component to force the height.
 
 <ImageSwitcher
   images={[
-    { src: "/img/foundations/size-base-spacing-50.svg", alt: "" },
-    { src: "/img/foundations/size-base-spacing-50-annotated.svg", alt: "" },
+    { src: "/img/foundations/size-nesting-base.png", alt: "" },
+    { src: "/img/foundations/size-nesting-base-annotated.png", alt: "" },
```

Comment:
Does the 'spacing key' need to be added within the image? From the guidance from Honey I assumed it was part of the site component. If its added in code can we show a spacing key for `--salt-spacing-50`?

Reply:
It's always been part of the image e.g.

![](https://www.saltdesignsystem.com/img/patterns/header-block/header-block-layout-spacing.png)

Reply:
Sorry that wasn't clear in the guide I received. Do you have time to update the images if I quickly add it to all of the relevant examples? It would also be good to export dark mode equivalents of each of them.

Reply:
I'll try find some time :)